src/features/swap/hooks/useIsExpiredSwap.ts

```diff
@@ -1,9 +1,6 @@
-import { useEffect, useState } from 'react'
+import { useEffect, useRef, useState } from 'react'
 import type { TransactionInfo } from '@safe-global/safe-gateway-typescript-sdk'
 import { isSwapOrderTxInfo } from '@/utils/transaction-guards'
-import useIntervalCounter from '@/hooks/useIntervalCounter'
```

Comment:
Does this mean that all other places where we use `useIntervalCounter` may cause an unnecessary re-render?